examples/avro-cli.py

```diff
@@ -57,21 +76,34 @@ def to_dict(self):
         }
 
 
-# Handle delivery reports served from producer.poll.
-def on_delivery(err, msg):
+"""
+    Handle delivery reports served from producer.poll.
+"""
+
+
+def on_delivery(err, msg, request):
     if err is not None:
-        print('Message delivery failed {}'.format(err))
+        print('Message {} delivery failed for user {} with error {}'.format(
+                                                                            request.correlation_id,
```

Comment:
I would use just `id`, since `correlation_id` might be confused with the Kafka protocol correlation id (and there isn't much correlation needed here since the object is passed.

Reply:
In my brain I was drawing a correlation between the response to my produce request with that object. Although I do pass in the object itself the response type be it error or success is tied to the request. So to me the id represents the relationship between those two. Flawed logic yes but I figured I'd explain myself anyway.